Cakefile

```diff
@@ -199,22 +199,36 @@ buildDocs = (watch = no) ->
   codeFor = require "./documentation/v#{majorVersion}/code.coffee"
 
   htmlFor = ->
-    marked = require 'marked'
-    markdownRenderer = new marked.Renderer()
-    markdownRenderer.heading = (text, level) ->
-      "<h#{level}>#{text}</h#{level}>" # Don’t let marked add an id
-    markdownRenderer.code = (code) ->
+    hljs = require 'highlight.js'
+    hljs.configure classPrefix: ''
+    markdownRenderer = require('markdown-it')
+      html: yes
+      typographer: yes
+      highlight: (str, lang) ->
+        # From https://github.com/markdown-it/markdown-it#syntax-highlighting
+        if lang and hljs.getLanguage(lang)
+          try
+            return hljs.highlight(lang, str).value
+          catch ex
+          return '' # No syntax highlighting
```

Comment:
Good catch! Probably didn’t matter because highlight.js never threw an exception, but can’t hurt to get it right.